make_profiler/dot_export.py

```
@@ -152,54 +155,56 @@ def export_dot(f, influences, dependencies, order_only, performance, indirect_in
-    f.write('cluster_inputs_DUMMY -> cluster_tools_DUMMY -> cluster_result_DUMMY [ style=invis ];')
+    dot.edge('cluster_inputs_DUMMY', 'cluster_tools_DUMMY', style='invis')
+    dot.edge('cluster_tools_DUMMY', 'cluster_result_DUMMY', style='invis')
 
     if 'cluster_not_implemented' in groups:
-        f.write('cluster_inputs_DUMMY -> cluster_not_implemented_DUMMY -> cluster_tools_DUMMY [ style=invis ];')
-        f.write('cluster_not_implemented_DUMMY -> cluster_order_only_DUMMY [ style=invis ];')
+        dot.edge('cluster_inputs_DUMMY', 'cluster_not_implemented_DUMMY', style='invis')
+        dot.edge('cluster_not_implemented_DUMMY', 'cluster_tools_DUMMY', style='invis')
+        dot.edge('cluster_not_implemented_DUMMY', 'cluster_order_only_DUMMY', style='invis')
 
     def format_deciminutes(k):
```

Ruff (0.11.9)

Missing return type annotation for private function `format_deciminutes`

Add return type annotation: `str`

(ANN202)